torch/nn/modules/module.py

```diff
@@ -25,15 +25,6 @@ def __repr__(self):
 
     __str__ = __repr__
 
-
-class ModuleAttributeError(AttributeError):
```

Comment:
Is there any BC concern by removing this? Or should we leave an alias here? @ezyang

Reply:
There might be users that caught the exception as `ModuleAttributeError`, so I'd personally say yes.

Reply:
I don't think it's ever mentioned anywhere in the docs and it's certainly not necessary to have anymore.
but someone definitely might be using it.

Reply:
@SsnL is correct, this change would be BC-breaking. @jbschlosser and I will have time to review tomorrow.

Reply:
We added this pretty recently Mar 26, 2020. I am supremely doubtful it is worth going through the rigamarole of preserving BC here.